usaspending_api/awards/models.py

```diff
@@ -102,9 +103,9 @@ class Award(DataSourceTrackedModel):
         ('L', 'Loan'),
     )
 
-    type = models.CharField(max_length=5, choices=AWARD_TYPES, verbose_name="Award Type")
+    type = models.CharField(max_length=5, choices=AWARD_TYPES, verbose_name="Award Type", null=True)
```

Comment:
Maybe a @alexjajabah or @playmakervi question, but I'm curious about allowing `nulls` here. If there's any potential at all that we'd aggregate data by this field (it's on Geoff's list), should we think about creating an "unknown" award type and use that instead `NULL`?

Again, not something that should hold us up now, but the general idea of `NULL` category-type fields might be something to put in the backlog.